ydb/core/ydb_convert/topic_description.h

```diff
@@ -0,0 +1,39 @@
+#pragma once
+
+#include <util/generic/fwd.h>
+
+namespace Ydb {
+    namespace Topic {
+        class Consumer;
+        class DescribeTopicResult;
+    }
+    class StatusIds;
+    enum StatusIds_StatusCode : int;
+}
+
+namespace NKikimrSchemeOp {
+    class TPersQueueGroupDescription;
+    class TDirEntry;
+}
+
+namespace NYql {
+    class TIssue;
+}
+
+namespace NActors {
+    struct TActorContext;
+}
+
+namespace NKikimrPQ {
+    class TPQTabletConfig_TConsumer;
+    class TPQConfig;
+}
+
+namespace NKikimr {
+
+bool FillConsumer(Ydb::Topic::Consumer& rr, const NKikimrPQ::TPQTabletConfig_TConsumer& consumer, Ydb::StatusIds_StatusCode& status, TString& error);
```

Comment:
```suggestion
bool FillConsumer(Ydb::Topic::Consumer& out, const NKikimrPQ::TPQTabletConfig_TConsumer& in, Ydb::StatusIds_StatusCode& status, TString& error);
```

Более понятный и консистентный нейминг.